tasks/parse-pdr/tests/test-parse-pdr.js

```diff
@@ -178,6 +186,26 @@ test.serial('parse-pdr properly parses PDR with granules of different data-types
   t.is(mod87MetFile.path, '/MODOPS/MODAPS/EDC/CUMULUS/FPROC/DATA');
   t.is(mod87MetFile.size, 44118);
   t.is(mod87MetFile.type, 'metadata');
+
+  // test MOD14 401
+  const mod14Granule = result.granules.find((granule) => granule.dataType === 'MOD14A1');
```

Comment:
Is the sole purpose of this to verify the `401` case?  

This is adding an additional test to the case `parse-pdr properly parses PDR with granules of different data-types` which should probably just be testing that the task can resolve multiple granules, not the specific resolution therein.  

e.g. 
```
test.serial('parse-pdr properly parses granule with version not prepended with zeros
```
(or some less terrible name)

Reply:
Yes, since the whole raison d'etre for issues with DATA_VERSION is the leading zeros in `006`, I thought it would be useful to add a *non*-leading zero case, to ensure that any changes we make don't break that case. 401 is an actual data version used for MODIS 4.1.

Reply:
I can see the argument for making a separate unit test and PDR, though.

Reply:
https://github.com/nasa/cumulus/pull/2030/commits/758952b38f1b017ef0fe4e27c8f9458e8407290b 👍